src/datastore/api.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+
+from ninja import Router, Query
+from ninja.pagination import paginate
+from django.views.decorators.csrf import csrf_exempt
+
+from .models import HistoricoAlerta
+from .schema import HistoricoAlertaSchema, HistoricoAlertaFilterSchema
+from registry.pagination import PagesPagination
+
+
+router = Router()
+
+paginator = PagesPagination
+paginator.max_per_page = 100
```

Comment:
I've manually set the max_per_page to 100 in this case, but more than that already feels too much for a single request